### PR TITLE
Bump version and fix version check

### PR DESCRIPTION
**Pull request by Anthchirp**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by Anthchirp** [comment]
@conda-forge-admin, please rerender

**Comment by Anthchirp** [comment]
@conda-forge-admin, please rerender

**Comment by Anthchirp** [comment]
Adding myself as maintainer as the PR has been untouched for a while. Feel free to undo.

@conda-forge-admin, please rerender

**Comment by Anthchirp** [comment]
@conda-forge/jellyfish this is ready for review/merging

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {% set name = "jellyfish" %}
-{% set version = "0.7.2" %}
+{% set version = "0.8.2" %}
 {% set bundle = "tar.gz" %}
-{% set hash = "cb09c50d7e2bb7b926fc7654762bc81f9c629e0c92ae7137bf22b34f39515286" %}
+{% set hash = "a499741401512d05bbd3556e448e960bc908eba3879fb73d450e8e91566a030b" %}
```
Comment (ocefpaf) [inline recipe/meta.yaml:4]: In the future I recommend using grayskull to regenerate the whole recipe and "modernize" all these jinja bits that could be troublesome.

Comment (Anthchirp) [reply recipe/meta.yaml:4]: yes, I've started doing that routinely now. This PR is from before that time 🙂

**Pull request merged**